test/rules.json

```
@@ -1305,6 +1316,29 @@
       "transformers": [
         "lowercase"
       ]
+    },
+    {
+      "id": "custom_action",
```

too confusing with an actual action ID
```suggestion
      "id": "custom_action_rule",
```